runlike/inspector.py

```diff
@@ -150,6 +150,14 @@ def parse_log(self):
                 log_options.add('--log-opt %s=%s' % (key, value))
         self.options += list(log_options)
 
+    def parse_extra_hosts(self):
+        hosts = self.get_fact("HostConfig.ExtraHosts") or []
+        hosts_options = set()
+        if hosts is not None:
+            for host_and_ip in hosts:
```

Comment:
I would ask to use this style instead:

```suggestion
            self.options += ['--add-host %s' % host for host in hosts]
```
instead of the whole block, since the set isn't really adding a lot, either (unless you have a running container with duplicate extra hosts, which seems unreasonable, and harmless, in any case).
WDYT?

Reply:
That looks much better. Thanks!